DebreGolgotaDibqGasha/DGDG001

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="NazMar001" xml:lang="en" type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>The 'Sane Golgota'  collection of  prayer book</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GA"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas:  Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="ETH0001DibqGasha"/>
+                        <idno>DGDG001</idno>
+                    </msIdentifier>
+                    <msContents>
+                          <msItem xml:id="ms_i1">
+                              <locus from="4r" to="50v"/>
+                              <title type="incomplete" ref="ETH0001SeneGol"/>
```

Comment:
Here, and below, I am not sure that the ID ETH0001SeneGol is a correct/existing one. In the title, you can (or should :)) refer to an internal ID of a literary work, for which you can search here (https://betamasaheft.eu/works/list) using different forms of a title. Maybe you mean LIT2246salotz? (https://betamasaheft.eu/works/LIT2246salotz/main)